File: openapi-generator/php-ze-ph/src/App/DTO/Tag.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class Tag
{
    /**
     * The addresses that belong to this tag.
     * @DTA\Data(field="addresses", nullable=true)
     * TODO check validator and strategy are correct and can handle container item type
     * @DTA\Strategy(name="ObjectArray", options={"type":\App\DTO\TaggedAddress::class})
     * @DTA\Validator(name="Collection", options={"validators":{
     *     {"name":"Dictionary", "options":{"type":\App\DTO\TaggedAddress::class}}
     * }})
     * @var \App\DTO\TaggedAddress[]
     */
    public $addresses;
    /**
     * The assets that belong to this tag.
     * @DTA\Data(field="assets", nullable=true)
     * TODO check validator and strategy are correct and can handle container item type
     * @DTA\Strategy(name="ObjectArray", options={"type":\App\DTO\TaggedAsset::class})
     * @DTA\Validator(name="Collection", options={"validators":{
     *     {"name":"Dictionary", "options":{"type":\App\DTO\TaggedAsset::class}}
     * }})
     * @var \App\DTO\TaggedAsset[]
     */
    public $assets;
    /**
     * The drivers that belong to this tag.
     * @DTA\Data(field="drivers", nullable=true)
     * TODO check validator and strategy are correct and can handle container item type
     * @DTA\Strategy(name="ObjectArray", options={"type":\App\DTO\TaggedDriver::class})
     * @DTA\Validator(name="Collection", options={"validators":{
     *     {"name":"Dictionary", "options":{"type":\App\DTO\TaggedDriver::class}}
     * }})
     * @var \App\DTO\TaggedDriver[]
     */
    public $drivers;
    /**
     * The GroupID that this tag belongs to.
     * @DTA\Data(field="groupId", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $group_id;
    /**
     * The ID of this tag.
     * @DTA\Data(field="id")
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $id;
    /**
     * The machines that belong to this tag.
     * @DTA\Data(field="machines", nullable=true)
     * TODO check validator and strategy are correct and can handle container item type
     * @DTA\Strategy(name="ObjectArray", options={"type":\App\DTO\TaggedMachine::class})
     * @DTA\Validator(name="Collection", options={"validators":{
     *     {"name":"Dictionary", "options":{"type":\App\DTO\TaggedMachine::class}}
     * }})
     * @var \App\DTO\TaggedMachine[]
     */
    public $machines;
    /**
     * Name of this tag.
     * @DTA\Data(field="name")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $name;
    /**
     * If this tag is part a hierarchical tag tree as a child tag, the parentTagId is the ID of this tag&#39;s parent tag.
     * @DTA\Data(field="parentTagId", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $parent_tag_id;
    /**
     * The sensors that belong to this tag.
     * @DTA\Data(field="sensors", nullable=true)
     * TODO check validator and strategy are correct and can handle container item type
     * @DTA\Strategy(name="ObjectArray", options={"type":\App\DTO\TaggedSensor::class})
     * @DTA\Validator(name="Collection", options={"validators":{
     *     {"name":"Dictionary", "options":{"type":\App\DTO\TaggedSensor::class}}
     * }})
     * @var \App\DTO\TaggedSensor[]
     */
    public $sensors;
    /**
     * The vehicles that belong to this tag.
     * @DTA\Data(field="vehicles", nullable=true)
     * TODO check validator and strategy are correct and can handle container item type
     * @DTA\Strategy(name="ObjectArray", options={"type":\App\DTO\TaggedVehicle::class})
     * @DTA\Validator(name="Collection", options={"validators":{
     *     {"name":"Dictionary", "options":{"type":\App\DTO\TaggedVehicle::class}}
     * }})
     * @var \App\DTO\TaggedVehicle[]
     */
    public $vehicles;
}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/Tag.scala

package org.openapitools.client.model


case class Tag (
    /* The addresses that belong to this tag. */
    _addresses: Option[List[TaggedAddress]],
    /* The assets that belong to this tag. */
    _assets: Option[List[TaggedAsset]],
    /* The drivers that belong to this tag. */
    _drivers: Option[List[TaggedDriver]],
    /* The GroupID that this tag belongs to. */
    _groupId: Option[Long],
    /* The ID of this tag. */
    _id: Long,
    /* The machines that belong to this tag. */
    _machines: Option[List[TaggedMachine]],
    /* Name of this tag. */
    _name: String,
    /* If this tag is part a hierarchical tag tree as a child tag, the parentTagId is the ID of this tag's parent tag. */
    _parentTagId: Option[Long],
    /* The sensors that belong to this tag. */
    _sensors: Option[List[TaggedSensor]],
    /* The vehicles that belong to this tag. */
    _vehicles: Option[List[TaggedVehicle]]
)
object Tag {
    def toStringBody(var_addresses: Object, var_assets: Object, var_drivers: Object, var_groupId: Object, var_id: Object, var_machines: Object, var_name: Object, var_parentTagId: Object, var_sensors: Object, var_vehicles: Object) =
        s"""
        | {
        | "addresses":$var_addresses,"assets":$var_assets,"drivers":$var_drivers,"groupId":$var_groupId,"id":$var_id,"machines":$var_machines,"name":$var_name,"parentTagId":$var_parentTagId,"sensors":$var_sensors,"vehicles":$var_vehicles
        | }
        """.stripMargin
}
